### ISSUE TITLE
Not correctly lint the module syntax of ES6.

### ISSUE THREAD

**Issue by Semigradsky** [palantir/tslint#354]
My code:
```javascript
import { XRegExp } from 'xregexp';
```

Lint errors:
```
src/index.ts[15, 10]: expected an assignment or function call
src/index.ts[15, 20]: expected an assignment or function call
src/index.ts[15, 25]: expected an assignment or function call
src/index.ts[15, 17]: missing semicolon
src/index.ts[15, 24]: missing semicolon
```

**Comment by adidahiya** [comment]
tracking upgrade to TS 1.5 here: #356
